### PR TITLE
Change assetStatusCards to the new assetStatusCards2 from info server

### PR DESCRIPTION
**Pull request by Jon-edge**
image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

https://github.com/EdgeApp/edge-info-server/pull/131

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207762282039357

### CODE REVIEW

**Review by swansontec — APPROVED** [review]

**Review by paullinator — CHANGES_REQUESTED** [review]
Do not merge. This PR is completely misses the spec of the task

**Review by paullinator — CHANGES_REQUESTED** [review]
Added a fixup but it's not tested. Please test and merge if it works

File: src/components/cards/InfoCardCarousel.tsx [hunk 1]
```diff
@@ -1,61 +1,44 @@
...
   navigation: NavigationBase
   screenWidth: number
+  // TODO: Add info server InfoCard export
+  cards?: PromoCard2[] | AssetStatus2[]
   countryCode?: string
 }
 
-export const PromoCards = (props: Props) => {
-  const { countryCode, navigation, screenWidth } = props
+export const InfoCardCarousel = (props: Props) => {
```
[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/cards/InfoCardCarousel.tsx:29]: I saw the diff worked out, or else I would have separated the commits. Will
default to separating to be safe in the future

File: src/components/cards/InfoCardCarousel.tsx [hunk 2]
```diff
@@ -1,61 +1,44 @@
...
+  const { countryCode, navigation, screenWidth, cards = [] } = props
```
[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/cards/InfoCardCarousel.tsx]: Didn't realize that can trigger an infinite loop. Tested behavior before and
after the fix and it checks out.

**Pull request merged**